jobs/mongodb_migration/src/mongodb_migration/migrations/_20230216141000_queue_split_names_from_streaming.py

```diff
@@ -0,0 +1,48 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2022 The HuggingFace Authors.
+
+import logging
+
+from libcommon.queue import Job
+from mongoengine.connection import get_db
+
+from mongodb_migration.check import check_documents
+from mongodb_migration.migration import Migration
+
+split_names = "/split-names"
+split_names_from_streaming = "/split-names-from-streaming"
+db_name = "queue"
+
+
+# connection already occurred in the main.py (caveat: we use globals)
+class MigrationQueueUpdateSplitNames(Migration):
+    def up(self) -> None:
+        logging.info(
+            f"Rename unicity_id field from Job[{split_names}][<dataset>][<config>][<split>] to"
+            f" Job[{split_names_from_streaming}][<dataset>][<config>][<split>]"
+        )
+
+        for job in Job.objects(type=split_names):
```

Comment:
I don't know if we should use mongoengine in the migration scripts since a mongoengine document is related to one version of the object, while here we are migrating from one version to the other. I tried to use pymongo directly. See also https://docs.mongoengine.org/guide/migration.html

Reply:
Also, maybe we could update both fields at the same time

Reply:
I created another PR to fix these typos and avoid using the documents https://github.com/huggingface/datasets-server/pull/837
One thing I couldn't do:  use functions/aggregations to replace the unicity_id in update_many function, I used the update/save instead